### PR TITLE
TSD 194 prim sec toggle

### PR DESCRIPTION
**Pull request by gkalvatcheva**
- responsive adjustments of new toggle
- footer fix for small screen sizes

### CODE REVIEW

File: app/i18n/locale_en.json [hunk 1]
```diff
@@ -13,7 +13,7 @@
     "tab.national": "National",
     "tab.national-view": "National View",
     "tab.school-view": "Schools",
-    "tab.school": "schools",
+    "tab.school": "Schools",
```
Comment (esoliani) [inline app/i18n/locale_en.json:19]: this should be done using style capitalize

File: app/styles/responsive.css [hunk 2]
```diff
@@ -1,138 +1,114 @@
-@media only screen and (max-width: 1280px) {
```
Comment (esoliani) [inline app/styles/responsive.css:1]: why are we version this file? and why are there errors on it?

File: app/i18n/locale_en.json (hunk 1 above)
Comment (uniphil) [inline app/i18n/locale_en.json:19]: I think it's ok -- we can `style: lowercase` if we need it lowered, which has less edge cases for CSS to mess up than `capitalize`.

File: app/styles/responsive.css (hunk 2 above)
Comment (uniphil) [inline app/styles/responsive.css:1]: good catch. removed.

**Pull request merged**